src/nanoemoji/write_font.py

```diff
@@ -272,11 +274,18 @@ def _update_paint_glyph(paint):
         if reuse_result is not None:
             # TODO: use the most compact valid transform
```

Comment:
I think this TODO is stale, since you have implemented the `transformed` function which is used immediately below. Shall I remove?

Reply:
I think so, yes